bindings/python/scripts/generate.py

```diff
@@ -1,105 +1,284 @@
-import json
+from context import scripts
+import scripts.utils as utils
 
 
-def read_json(filename):
-    with open(filename, "r") as f:
-        return json.load(f)
+class bind:
+    def __init__(self, root):
+        self._state_stack = []
+        self.linelist = []
```

Comment:
Segregate public and private variables (without and with a beginning underscore)

Also please add documentation for public members